src/components/controls/Slider.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import Slider from '@smui/slider';
+  import { Writable } from "svelte/store"
+  import { onMount, onDestroy } from 'svelte';
+  import type { ControlProperty, LayerProperty, SliderProperties } from '../../config';
+  import Control from './Control.svelte';
+  import { GetOrCreateControlStore } from '../../stores';
+
+  export let layerProperty: LayerProperty;
+  export let controlProperty: ControlProperty;
+  let args: SliderProperties;
+
+  let store: Writable;
+  let value: number;
+
+  onMount(() => {
+    // FIXME move this function somewhere else
+    const id = `${layerProperty.id}.${controlProperty.type}`;
+    store = GetOrCreateControlStore(id);
+    args = controlProperty.args as SliderProperties;
+    value = args.initialValue;
+  });
+
+  $: {
+    if (args !== undefined) {
```

Comment:
Things like this are until I learn more about svelte life-cycle. It's a work in progress. I can put a FIXME here.